.npmrc

```diff
@@ -0,0 +1 @@
+registry = https://registry.npmjs.com/
```

Comment:
What the FUCK is this?

Reply:
Oh right, I have a global default NPM registry on my laptop that I need to override. 

I say it's fine to make the decision that all our components will come from registry.npmjs.com.